### PR TITLE
Add recipe to audit servers with Deno

### PR DESCRIPTION
**Pull request by spawnia**
I tried to setup audit tests as described in the Jest example, but found the setup of Jest itself quite cumbersome. Thus, I tried to see if Deno was simpler.

~Unfortunately, the example does not work as of now. You can try writing the example code to a file `test.ts` and run it with `deno test test.ts`. I am currently getting the following error:~

```
error: Uncaught SyntaxError: The requested module '/v111/graphql@16.6.0/deno/graphql.js' does not provide an export named 'GraphQLError'
    at <anonymous> (https://esm.sh/v111/graphql-http@1.16.0/deno/graphql-http.js:2:90)
```

I found that using [npm specifiers](https://deno.land/manual@v1.31.3/node/npm_specifiers#npm-specifiers) fixes the above mentioned error with the missing export.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -736,6 +736,33 @@ for (const audit of serverAudits({
...
+<summary><a href="#audit-deno">🔗</a> Audit for servers usage in <a href="https://deno.land">Deno</a> environment</summary>
+
+```ts
+import { serverAudits } from 'npm:graphql-http';
+
+for (const audit of serverAudits({
+  url: 'http://localhost:4000/graphql',
+  fetchFn: fetch,
+})) {
+  Deno.test(audit.name, async () => {
```
Comment (spawnia) [inline README.md:749]: I found that using [npm specifiers](https://deno.land/manual@v1.31.3/node/npm_specifiers#npm-specifiers) fixes the above mentioned error with the missing export. However, it introduces another problem - the implementation of the audits leaks resources.

```
SHOULD use 400 status code on JSON parsing failure when accepting application/graphql-response+json => ./test.ts:9:8
error: AssertionError: Test case is leaking 1 resource:

 - A fetch response body (rid 345) was created during the test, but not consumed during the test. Consume or close the response body `ReadableStream`, e.g `await resp.text()` or `await resp.body.cancel()`.

    at assert (ext:deno_web/00_infra.js:353:11)
    at resourceSanitizer (ext:cli/40_testing.js:417:5)
    at async Object.exitSanitizer [as fn] (ext:cli/40_testing.js:435:7)
    at async runTest (ext:cli/40_testing.js:840:5)
    at async runTests (ext:cli/40_testing.js:1098:20)
```

Changing this line to `Deno.test(audit.name, { sanitizeResources: false },async () => {` makes the test pass, but I am wondering if the audits could be modified to not leak resources.

Comment (enisdenjo) [reply README.md:749]: I guess the cloned response is whats considered leaking, if this is the case - there's nothing we can do - the cloned response is necessary for audit users to have access to the response for inspection.

Is this the case? How can we check?

**Comment by enisdenjo** [comment]
Weird, v16.6.0 on esm.sh indeed does not have the necessary export: https://esm.sh/v111/graphql@16.6.0/deno/graphql.js.

However, v16.5.0 has it: https://esm.sh/v111/graphql@16.5.0/deno/graphql.js.

What changed? 🤔

**Review by enisdenjo — APPROVED** [review]
I'd say [this](https://github.com/graphql/graphql-http/pull/63#issue-1634061177) has something to do with graphql-js itself, as noticed [here](https://github.com/graphql/graphql-http/pull/63#issuecomment-1478304109).

With this in mind, I think we're good to merge this - thank you!

**Pull request merged**